examples/phone-chatbot/bot_daily_gemini.py

```diff
@@ -164,20 +175,10 @@ async def main(
     detect_voicemail: bool,
     dialout_number: Optional[str],
 ):
-    # dialin_settings are only needed if Daily's SIP URI is used
-    # If you are handling this via Twilio, Telnyx, set this to None
-    # and handle call-forwarding when on_dialin_ready fires.
-
-    # We don't want to specify dial-in settings if we're not dialing in
     dialin_settings = None
-    if callId and callDomain:
+    if callId != "None" and callDomain != "None":
```

Comment:
I think what you probably want is to make the arguments optional. That is:

```
async def main(....., callId: Optional[str], callDomain: Optional[str], ...)
```

and then you can keep doing:

```
    if callId and callDomain:
```